protocol/persistence.go

```diff
@@ -727,6 +727,12 @@ func (db sqlitePersistence) Contacts() ([]*Contact, error) {
 			}
 			contact.SocialLinks = append(contact.SocialLinks, link)
 		}
+
+		profileShowcase, err := db.GetProfileShowcaseForContact(contact.ID)
```

Comment:
this probably should be a join for performance reasons? (same with social links) at least we could do 
(1 for contacts, one for social links, one for profile showcase), instead of now that we do`2*nContacts` + 1 query, I think that's going to be slow maybe?

Reply:
The thing is, both for social links and profile entries, there are several rows for one contact id, so `LEFT JOIN` will trigger a whole query to have several rows. But challenge to optimise this func is interesting. Can I try to do it in separated PR ?

Reply:
sure, that's no problem

Reply:
https://github.com/status-im/status-go/issues/4239